### DISCUSSION TITLE
Will specmatic be supporting OpenAPI workflows

### DISCUSSION THREAD

**Discussion by prateekkarmakar** [specmatic/specmatic#1510] [Ask For Help] [answered]
https://swagger.io/blog/meet-the-new-api-workflows-specification/ -> Does specmatic plan to support the following "OpenAPI workflow specification is to describe, in a machine-readable format, a chain of API requests. It is no longer about sending some input and receiving an output. It’s about representing a sequence of API calls that collectively fulfil a business process."

**Comment by harikrishnan83** [comment, marked as answer]
hello @prateekkarmakar , yes we are already working on adding support for [Arazzo](https://www.openapis.org/arazzo) (the "[Workflows Specification](https://github.com/OAI/Arazzo-Specification/blob/main/versions/1.0.0.md)" mentioned in the link you have shared above) and this should be available in Q1 2025. Please refer to our [roadmap](https://specmatic.io/roadmap) and [feature overview](https://specmatic.io/roadmap/#feature-overview) for more details. Thanks.

**Reply by prateekkarmakar** [reply]
Thanks for the details
